css/properties/counter-increment.json

```diff
@@ -30,19 +30,19 @@
               "version_added": "9.2"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "10"
             },
             "safari": {
-              "version_added": "3"
+              "version_added": "3.1"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "3.2"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": true
             },
             "webview_android": {
-              "version_added": null
+              "version_added": "2"
```

Comment:
This should be `true`. Webview didn't become a Chromium fork until 37. We don't have any information about pre-chromium webviews.

Reply:
Whoops!